run_tests.py

```diff
@@ -1,10 +0,0 @@
-import os
```

Comment:
Removed this file because I found the make file made it a bit redundant. Happy to revert if people prefer it though